src/main/java/org/myrobotlab/service/Python.java

```diff
@@ -475,7 +477,7 @@ public boolean exec(String code) {
    *         false if there was an exception.
    */
   public boolean exec(String code, boolean blocking) {
-    log.info("exec(String) \n{}", code);
+    log.debug("exec(String) \n{}", code);

```

Comment:
having this at info level is really handy for no-worky's...

Reply:
yup

Reply:
updated